README.mediawiki

```diff
@@ -448,13 +448,13 @@ Those proposing changes should consider that ultimately consent may rest with th
 | Pavol Rusnak
 | Standard
 | Final
-|-
+|- style="background-color: #cfffcf"
 | [[bip-0085.mediawiki|85]]
 | Applications
 | Deterministic Entropy From BIP32 Keychains
 | Ethan Kosakovsky
 | Informational
-| Draft
+| Final
```

Comment:
Why Final and not Proposed per [this comment](https://github.com/bitcoin/bips/pull/1600#issuecomment-2394691135)?

Reply:
comment states "propsed or final", this BIP should be final as it is widely deployed and used by many projects.